--- src/main/java/org/vafer/jdependency/utils/DependencyUtils.java
+++ src/main/java/org/vafer/jdependency/utils/DependencyUtils.java
@@ -54,7 +54,7 @@
 
                 if (name.endsWith(".class")) {
                     final DependenciesClassAdapter v = new DependenciesClassAdapter();
-                    new ClassReader( inputStream ).accept( v, 0 );
+                    new ClassReader( inputStream ).accept( v, ClassReader.EXPAND_FRAMES );
                     dependencies.addAll(v.getDependencies());
                 } else {
                     IOUtils.copy(inputStream, nullStream);
